backend/controllers/prisonerProfile/prisonerSentenceAndRelease.js

```diff
@@ -4,15 +4,17 @@ const courtCasesViewModel = require('./sentenceAndReleaseViewModels/courtCasesVi
 const { serviceUnavailableMessage } = require('../../common-messages')
 const { readableDateFormat } = require('../../utils')
 
-module.exports = ({ prisonerProfileService, elite2Api, logError }) => async (req, res) => {
+module.exports = ({ prisonerProfileService, elite2Api, systemOauthClient, logError }) => async (req, res) => {
   const { offenderNo } = req.params
 
   try {
+    const systemContext = await systemOauthClient.getClientCredentialsTokens()
```

Comment:
could possibly pass through the users username so it appears as part of the audit?

Reply:
That's a good point actually. We use client creds in a few places and don't pass the username, and I agree it should.